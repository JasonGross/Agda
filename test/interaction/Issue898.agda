module Issue898 where

id : {A : Set} -> A -> A
id = ?
