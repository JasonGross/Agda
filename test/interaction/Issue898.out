Agda2> (agda2-status-action "")
(agda2-info-action "*Type-checking*" "" nil)
(agda2-highlight-clear)
(agda2-info-action "*Type-checking*" "Checking Issue898 (Issue898.agda).\n" t)
(agda2-info-action "*Type-checking*" "Finished Issue898.\n" t)
(agda2-status-action "")
(agda2-info-action "*All Goals*" "?0 : {A : Set} → A → A\n" nil)
((last . 1) . (agda2-goals-action '(0)))
Agda2> (agda2-give-action 0 "λ { {A} x → x }")
(agda2-status-action "")
(agda2-info-action "*All Goals*" "" nil)
((last . 1) . (agda2-goals-action '()))
Agda2> 